test/bigtest/tests/sets/create-edit-test.js

```diff
@@ -161,14 +270,21 @@ describe('Sets', () => {
       id: 'id',
       name: 'initial values name',
       description: 'initial values description',
+      filteringConditions: [{
+        name : 'location',
+        active: true,
+        value : 'location 1',
+        setSpec : 'Loc_L1',
+      }],
     };
     const currentPath = `/settings/oai-pmh/sets/${initialValues.id}/edit`;
     const redirectBackPath = `/settings/oai-pmh/sets/${initialValues.id}/view`;
 
-    beforeEach(function () {
+    beforeEach(async function () {
       this.server.create('set', initialValues);
 
       this.visit(currentPath);
+      await wait(300);
```

Comment:
The preferred pattern to to await the presence of an expected element when the page is loaded. That way once the element is present, you can proceed which will likely be faster than 300ms. It is also more resilient if the load takes longer than 300ms so you don't need to do trial-and-error with arbitrary wait times.

Reply:
fixed